src/sm_bluesky/common/plan_stubs/motions.py

```diff
@@ -19,7 +20,7 @@ class MotorTable(RootModel):
 
 @plan
 def move_motor_with_look_up(
-    slit: Motor,
+    slit: Motor | SimMotorExtra,
```

Comment:
Nit:  I agree this is the most technically accurate, I'd prefer to keep the current typing for the plan. Since the strict typing/implementation detail is not visible or useful to the user,
We can use the actually typing for the test like this as XYZStage/Motor has the same API as the SimStage/SimMotor.
```python
 def test_motor_with_look_up_fail(
    run_engine: RunEngine, sim_motor_step: XYZStage
) -> None:
```

```suggestion
    slit: Motor 
```